pairs.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>

```

Comment:
File names should reflect the intent or the content.
 There is a scope for improvement here by keeping test code and production code in different files - this helps in readability and maintainability.